src/components/UserNameInputDialog.vue

```diff
@@ -1,65 +1,93 @@
 <template>
   <InteractionDialog
     variant="text-only"
-    :show-dialog="showDialog"
-    title="Switch user"
-    :actions="interactionDialogActions"
+    persistent
+    :show-dialog="showUserDialog"
+    title="Manage users"
```

Comment:
```suggestion
    title="Manage users on: {{ currentVehicleName }}"
```
Is this possible, instead of having the current vehicle on a separate line?